core/vtk/ttkArrayPreconditioning/ttkArrayPreconditioning.cpp

```diff
@@ -1,3 +1,4 @@
+#include "DataTypes.h"
```

Comment:
please use brackets (i.e. `<` and `>` instead of quotes). this would yield missing header errors if that code is directly called by an external program.

Reply:
This line was added automatically by my IDE, but is actually useless here.
I deleted it.